backend/controllers/getGroups.test.js

```diff
@@ -8,8 +8,9 @@ describe('select groups factory', () => {
     it('should call groups render', async () => {
       getGroups.mockResolvedValue([{ groupName: 'name', groupCode: 'code' }])
 
+      const req = { flash: jest.fn() }
```

Comment:
would be good to test that `flash` is called by the `render`

Reply:
added